2025/shell_scripting/Week 3 - Shell Scripting Challenge/user_management.sh

```
@@ -0,0 +1,76 @@
+#create user using shell script
+
+
+#!/bin/bash
```

Shellcheck (0.10.0)

[error] The shebang must be on the first line. Delete blanks and move comments.

(SC1128)